<ui version="4.0" >
 <class>InternetExtensionDlg</class>
 <widget class="QWidget" name="InternetExtensionDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="gbTexts" >
     <property name="title" >
      <string>Texte</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="lbTextDownloadURL" >
        <property name="text" >
         <string>Textdownload-URL:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="leTextUpdateURL" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Wiktionary-Import</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="lbDumpOverview" >
          <property name="text" >
           <string>Dump Ãœbersicht:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="leWikiDumpUrls" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="lbDumpPrefix" >
          <property name="text" >
           <string>Dump Prefix:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="leWikiPrefix" />
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="lbDumpPostfix" >
          <property name="text" >
           <string>Dump Postfix:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="leWikiPostfix" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lbExample" >
        <property name="text" >
         <string>Beispiel:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbExampleURL" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
